src/translations/nbTranslations.json

```
@@ -2100,7 +2100,7 @@
       "previous_candidate": "Gå til forrige kandidat",
       "publication_points": "Publiseringspoeng",
       "reject_nvi_candidate": "Avvis resultat",
-      "reject_nvi_candidate_description": "Klikk på knappen \"{{buttonText}}\" dersom forskningsresultatet ikke skal rapporteres. Du må oppgi en grunn for avvisningen.",
+      "reject_nvi_candidate_description": "Trykk på \"{{buttonText}}\" om publikasjonen ikke kan rapporteres. Du må begrunne avisingen.",
```

Vært på flytur i det siste? :D 
```suggestion
      "reject_nvi_candidate_description": "Trykk på \"{{buttonText}}\" om publikasjonen ikke kan rapporteres. Du må begrunne avvisningen.",
```